app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -185,6 +189,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * get format of translation
+     * @return
+     */
+    public TranslationFormat getFormat() {
+        return mTranslationFormat;
+    }
+
+    private TranslationFormat readTranslationFormat() {
+        String formatStr = manifest.getString("format");
+        TranslationFormat format = TranslationFormat.get(formatStr);
+        if(null == format) {
```

Comment:
Ah good point. However, that could produce problems later on because are are placing legacy/migration specific information directly in the object code. I think it would be safer to assume a perfect world and follow the current manifest spec strictly. If we actually ran into an issue where some USX code made it out then our migration code would have been at fault not this method.

Let's depend on on your usx to usfm migration code to correctly update everything and set the format.